specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+              "$ref": "#/definitions/DynamicPoolConfiguration"
+            },
+            "customContainerTemplate": {
+              "description": "The custom container configuration if the containerType is CustomContainer.",
+              "$ref": "#/definitions/CustomContainerTemplate"
+            },
+            "sessionNetworkConfiguration": {
+              "description": "The network configuration of the sessions in the session pool.",
+              "$ref": "#/definitions/SessionNetworkConfiguration"
+            },
+            "poolManagementEndpoint": {
+              "description": "The endpoint to manage the pool.",
+              "type": "string",
+              "readOnly": true
+            },
+            "provisioningState": {
+              "description": "Provisioning state of the session pool.",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled",
+                "Deleting"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "SessionPoolProvisioningState",
+                "modelAsString": true
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SessionPoolUpdatableProperties": {
+      "description": "Container App session pool updatable properties.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Session pool resource specific updatable properties.",
+          "type": "object",
+          "properties": {
+            "scaleConfiguration": {
+              "description": "The scale configuration of the session pool.",
+              "$ref": "#/definitions/ScaleConfiguration"
+            },
+            "secrets": {
+              "description": "The secrets of the session pool.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Secret"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "dynamicPoolConfiguration": {
+              "description": "The pool configuration if the poolManagementType is dynamic.",
+              "$ref": "#/definitions/DynamicPoolConfiguration"
+            },
+            "customContainerTemplate": {
+              "description": "The custom container configuration if the containerType is CustomContainer.",
+              "$ref": "#/definitions/CustomContainerTemplate"
+            },
+            "sessionNetworkConfiguration": {
+              "description": "The network configuration of the sessions in the session pool.",
+              "$ref": "#/definitions/SessionNetworkConfiguration"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SessionPoolCollection": {
+      "description": "Session pool collection ARM resource.",
```

Comment:
>ARM [](http://example.com/codeflow?start=46&length=3)

Please change it to not mention 'ARM'. You can use the term 'Azure'.

ARM is implementation detail, not something we need to doc.

Reply:
Fixed